docs/source/references/config.md

```diff
@@ -49,7 +47,8 @@ The following is the default configuration used by Iroha.
     "MAX_TRANSACTIONS_IN_BLOCK": 512,
     "ACTOR_CHANNEL_CAPACITY": 100,
     "GOSSIP_BATCH_SIZE": 500,
-    "GOSSIP_PERIOD_MS": 1000
+    "GOSSIP_PERIOD_MS": 1000,
+    "DEBUG_FORCE_SOFT_FORK": false
```

Comment:
Looks like you generated `config.md` with `debug` profile. Can you please recreate it with `release`, because we don't expose `"DEBUG_FORCE_SOFT_FORK"` parameter.

Reply:
Yup, I fixed it already.